### DISCUSSION TITLE
How To Add Image but not enter the URL of the image?

### DISCUSSION THREAD

**Discussion by JenuelDev** [ueberdosis/tiptap#854] [General Questions] [answered]
I like first to upload the image to my server that returns the URL of the image, then add the URL to the command that will set the image in the editor.

How to Do that? thanks, guys 👍🏻

**Comment by hanspagel** [comment, marked as answer]
Hope those two links give you a head start:

https://github.com/ueberdosis/tiptap/issues/89#issuecomment-443138315
https://gist.github.com/slava-vishnyakov/16076dff1a77ddaca93c4bccd4ec4521

✌️

**Reply by JenuelDev** [reply]
thanks, after reading it... and try and error, I was able to pull it out.. thanks thanks 👍🏻
